.github/workflows/python-app.yml

```diff
@@ -19,6 +19,10 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: 3.7
+      - uses: actions/cache@v2
```

Comment:
let's give the step a name so we know what's being cached

Reply:
Good suggestion! Added in the latest commit.